contracts/contracts/token/WOETH.sol

```diff
@@ -28,8 +29,15 @@ import { OETH } from "./OETH.sol";
 contract WOETH is ERC4626, Governable, Initializable {
```

Comment:
Some of the comment above this is no longer accurate.

Reply:
Fixed: https://github.com/OriginProtocol/origin-dollar/pull/2453/commits/599c41524ca3698056f12515b2fd1ba1c448f376